checkout-ui-custom/src/partials/Deliver/AddAddressManualForm.js

```diff
@@ -0,0 +1,258 @@
+import { isValidNumber } from 'libphonenumber-js';
+import { formatPhoneNumber } from '../../utils/phoneFields';
+import { addOrUpdateAddress } from '../../utils/services';
+import setAddress from '../../utils/setAddress';
+import FormField from './Elements/FormField';
+import { getBestRecipient } from './utils';
+
+export const ADD_FORM_RECEIVER_PHONE_ID = 'bash--input-add-address-form-receiverPhone';
+
+const Heading = () => /* html */ `
+    <div class="bash--heading">
+      <h3>Add a new delivery address</h3>
+      <a href="#" class="back-button--select" data-view="select-address">&lt; Back</a>
+    </div>
+`;
+
+const SaveButton = () => /* html */ `
+  <button
+    class="submit btn-go-to-payment btn btn-large btn-success"
+    id="btn-save-address"
+    type="submit">
+    Save Address
+  </button>
+`;
+
+const AddAddressManualForm = () => {
+  const fields = [
+    {
+      name: 'addressId',
+      type: 'hidden',
+      value: '',
+      required: false,
+    },
+    {
+      name: 'addressName',
+      type: 'hidden',
+      value: '',
+      required: false,
+      maxLength: 50,
+    },
+    {
+      name: 'lat',
+      required: false,
+      type: 'hidden',
+      value: '',
+    },
+    {
+      name: 'lng',
+      required: false,
+      type: 'hidden',
+      value: '',
+    },
+    {
+      name: 'number',
```

Comment:
<img width="410" alt="image" src="https://github.com/user-attachments/assets/ea1ec729-1d72-46eb-bbe6-2e47ac08a7c0">

no seperation between street and number in designs 

https://www.figma.com/design/M9vAdnEoGENzSD3OkaE7BB/%5BWEB%5D-Cart-%26-Checkout?node-id=901-244&t=VhMljcQxaMHMRNrz-0